api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/InitializerImportExportTest.kt

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2022 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.inference.keras
+
+import org.jetbrains.kotlinx.dl.api.core.Sequential
+import org.jetbrains.kotlinx.dl.api.core.initializer.*
+import org.jetbrains.kotlinx.dl.api.core.layer.core.Dense
+import org.jetbrains.kotlinx.dl.api.core.layer.core.Input
+import org.junit.jupiter.api.Test
+
+class InitializerImportExportTest {
```

Comment:
One optional enhancement that comes to mind is to add tests comparing outputs of original models and deserialized ones.